pom.xml

```diff
@@ -126,6 +131,18 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>1.10.19</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.scijava</groupId>
+			<artifactId>scijava-ui-swing</artifactId>
```

Comment:
This is a circular dependency: `scijava-ui-swing` depends on `scijava-common`. Could you please, rather than mocking a `SwingSDIUI`, just implement your own `UserInterface` plugin in the test class itself as a static inner class, and then use that? Then you wouldn't need Mockito either.